js/ui/calendar/ui.calendar.base_view.js

```diff
@@ -198,12 +205,29 @@ const BaseView = Widget.inherit({
                 });
             }
         }));
+
+        if(this.option('selectionMode') === 'range') {
+            this._createCellHoverAction();
+            eventsEngine.off(this._$table, CALENDAR_DXHOVER_EVENT_NAME);
```

Comment:
Do we may concrete what hover event using in variable name? 

CALENDAR_DXHOVERSTART_EVENT_NAME
I think it gives us more information about event.

Reply:
fair point, updated this